yarn.lock

```diff
@@ -12636,6 +12636,11 @@ ip@^1.1.5:
   resolved "https://registry.yarnpkg.com/ip/-/ip-1.1.5.tgz#bdded70114290828c0a039e72ef25f5aaec4354a"
   integrity sha1-vd7XARQpCCjAoDnnLvJfWq7ENUo=
 
+ip@^2.0.0:
```

Comment:
interesting, it used to require ip ^1
now it uses IP ^2 which is why this block exists, however, I don't see a block for 1p@^1...?
https://github.com/adobe/react-spectrum/pull/3195/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR4193

Reply:
I can try regenerating the lockfile entirely